specs/ukhpi-data-display.spec

```diff
@@ -0,0 +1,118 @@
+* User can expand a section to view a chart of 'House price index by buyer status'
+
+## Chart sections - Funding status
+
+Tags: data, results
+
+* User can view section heading for 'Funding status' and click quick link at top of page to anchor to section
+* User can expand a section to view a chart of 'Average price by funding status'
+* User can expand a section to view a chart of 'Percentage change (yearly) by funding status'
+* User can expand a section to view a chart of 'Percentage change (monthly) by funding status'
+* User can expand a section to view a chart of 'House price index by funding status'
+* User can expand a section to view a chart of 'Sales volume by funding status'
+
+## Chart sections - Property status
+
+Tags: data, results
+
+* User can view section heading for 'Property status' and click quick link at top of page to anchor to section
+* User can expand a section to view a chart of 'Average price by property status'
+* User can expand a section to view a chart of 'Percentage change (yearly) by property status'
+* User can expand a section to view a chart of 'Percentage change (monthly) by property status'
+* User can expand a section to view a chart of 'House price index by property status'
+* User can expand a section to view a chart of 'Sales volume by property status'
+
+## View data graph tab
+
+Tags: data, results
+
+* User can expand first item in first section to view a chart of 'Average price type of property'
+* User can see tabbed navigation which defaults to 'See data graph' tab content
+* User can see a line graph of data
+* The line graph defaults to showing 'All property types'
+* The User can select property type checkboxes to add data points to line graph
+* The User can click the magnifying glass icon to view the line graph in a modal dialogue
+
+## View data table tab
+
+Tags: data, results
+
+* Default query is set to 'United Kingdom' and 'October 2020 to October 2021'
+* Click to expand the view of 'Average price by type of property'
+* Clicking 'See data table' tab shows tabular data of query
+* Table row display for 'Date' 'Oct 2020' value is '£243.821'
+* The User can select property type checkboxes to add data columns to the table
+* Clicking 'print this table' will open a new browser window showing a printable view of data
+* The print control will open
+
+## Download this data tab
+
+Tags: data, results
+
+* Default query is set to 'United Kingdom' and 'October 2020 to October 2021'
+* Click to expand the view of 'Average price by type of property'
+* Clicking 'Download this data' tab shows a selection of download links to view data exports
+* Clicking the 'download CSV/spreadsheet' button will prompt a CSV file download
+* Clicking the 'download JSON' button will prompt a JSON download
+* Clicking the 'try the SPARQL query' link will open the SPARQL console with the query pre-populated
+
+## Compare with another location tab
```

Comment:
The comparison view is a separate page in the app, with its own flow once you get into it. I wonder if it makes sense to fork these checks into a separate `.spec` file?